plugin/src/main/kotlin/org/jlleitschuh/gradle/ktlint/BaseKtlintCheckTask.kt

```diff
@@ -207,7 +207,8 @@ abstract class BaseKtlintCheckTask(
                 }
             additionalConfig(argsWriter)
             filesToCheck.forEach {
-                argsWriter.println("\"${it.toRelativeFile()}\"")
+                val filePath = it.toRelativeFile().path.replace("\\", "\\\\")
```

Comment:
Shouldn't this be `replaceAll`? Doesn't `replace` only replace the first occurance?

Reply:
Kotlin does not have `replaceAll`. `replace` function replaces all occurrences.